addons/compat_spe/compat_spe_csw/CfgVehicles.hpp

```diff
@@ -0,0 +1,240 @@
+class CfgVehicles {
+    class All {
+        class EventHandlers;
+    };
+    class AllVehicles: All {};
+    class Land: AllVehicles {};
+    class LandVehicle: Land {};
+    class StaticWeapon: LandVehicle {
+        class ACE_Actions {
+            class ACE_MainActions;
+        };
+        class Turrets {
+            class MainTurret;
+        };
+        class UserActions;
+    };
+
+    // --- Mortars -----------------------------------------------------------------
+    class StaticMortar: StaticWeapon {
+        class Turrets: Turrets {
+            class MainTurret: MainTurret {};
+        };
+    };
+    class SPE_StaticMortar_base: StaticMortar {
+        class Turrets: Turrets {
+            class MainTurret: MainTurret {};
+        };
+        class EventHandlers: EventHandlers {
+            class SPE_StaticWeaponsHandler {
+                init = "";
+            };
+        };
+    };
+
+    class SPE_US_Mortar_base: SPE_StaticMortar_base {};
+    class SPE_M1_81: SPE_US_Mortar_base {
+        class Turrets: Turrets {
+            class MainTurret: MainTurret {
+                magazines[] = {};
```

Comment:
Why we're zeroing magazines array here? Won't this make the mortar unusable/empty, CSW compat does not need this IIRC?

Reply:
Yeah, CSW shouldn't need this

Reply:
csw was designed to be a runtime setting
so we shouldn't be doing this or fully removing actions (but we might have to modify them to be compat with csw)
```
        class UserActions {
            delete Prepare_Illu_Selected;
```
maybe we just comment out
```
#include "CfgVehicles.hpp"
#include "CfgWeapons.hpp"
```
for now and finish this sub-component up later